### ISSUE TITLE
Find mechanism for cleanup in long-running server setups

### ISSUE THREAD

**Issue by christianzoellner** [hpi-sam/digital-fuesim-manv#183]
It is possible to delete any exercise with trainer id id using DELETE /api/exercise/id, and there is #179 to find a way to invoke this with the frontend.

However, there sould also be an automatic deletion/achtiving mechanism, unless you can convince me that database / persistance https://github.com/hpi-sam/BP2021HG1/commit/f8dfcbc7ec26e744d541414f861142a16785b984 will solve this implicitly.

This is not so high priority, but it should be done before a must-have version is delivered to our customers (and run long-term on servers).

**Comment by ClFeSc** [comment]
@christianzoellner I just did a quick test: An empty database, i.e. only filled with the tables, not with actual user data, takes up about 10 MB. When importing the exercise from the BABZ, the database takes up about 20 MB, indicating that the exercise itself takes up about 10 MB. I'd say it's acceptable, as servers usually have many GBs to spend. Do you agree?

**Comment by christianzoellner** [comment]
@ClFeSc 10MB per exercise is acceptable for any deployment we might have in the foreseeable future. So there is an argument that this issue has a low priority for now. 

However, 10MB per exercise can scale quickly if there are a dozen exercises per day, so this issue will come back at some point. Also, infinite logging might become a GDPR-problem depending on the specific deployment.

For now, I think it's fine that you do not implement any automatic cleaning in the last few days. But you should at least document this issue of an infinitely logging database somewhere.
